tensorflow_io/kafka/python/ops/kafka_dataset_ops.py

```diff
@@ -44,6 +46,17 @@ def __init__(self,
       eof: If True, the kafka reader will stop on EOF.
       timeout: The timeout value for the Kafka Consumer to wait
                (in millisecond).
+      global_cfg: A `tf.string` tensor containing global configuration
+                  properties in [Key=Value] format,
+                  eg. ["enable.auto.commit=false",
+                       "heartbeat.interval.ms=2000"],
+                  please refer to 'Global configuration properties'
+                  in librdkafka doc.
+      topic_cfg: A `tf.string` tensor containing topic configuration
+                 properties in [Key=Value] format,
+                 eg. ["auto.offset.reset=earliest"],
+                 please refer to 'Topic configuration properties'
+                 in librdkafka doc.
```

Comment:
@zhjunqin Maybe it makes sense to change the arg names to `config_global` and `config_topic`, so that they could be separated from the configuration parameters that belongs to generic Dataset?

Reply:
OK, I will change the name. : )